.golangci.yaml

```diff
@@ -0,0 +1,24 @@
+run:
```

Comment:
What does it do if this file isn't present?

Reply:
run with the default settings, but this is just the action in serving anyway. It won't propagate. I've create a different action that shall be propagated through that'll key off of the existence of this file.